contrib/extract_image_name.sh

```diff
@@ -0,0 +1,35 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+# Extracts the image name (repo:tag) from the tarball without running it
+# Does not require docker to be installed
+
+mkdir temp_for_extracting_id
+cd temp_for_extracting_id
```

Comment:
Why do you need to do this in a subdirectory?

Reply:
Originally this was for easy cleanup, and to avoid accidentally overriding files. I just made a change that makes cleanup easy regardless. Should I not be concerned about overriding files? If so, I can remove the subdirectory usage.

Reply:
Sounds good, fixed.